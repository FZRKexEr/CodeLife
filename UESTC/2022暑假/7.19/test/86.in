10
nlenfjgkthsscvfxpgegzfmyxuxnbtuomrvblupkscstausqyiomdgahjgmusgahlrphscrqzzuahxtvoncqrbgzgxushkvvpfuw
7
ctiapprwribjsthrkkunhibmrglsmrapcwcubrdhnqhungxjhqxwoxprrmhldrkgledsgcekgsjyqnjhxezlbvpewyuaqxanqpyn
7
tcvtkgjpqxiiwatdqzcswtzyapfesudrhfyelnyhvfkywavszquksppzmwqrrdsiywmdwufjyalxjnrhlcbkvqvjegjzzqxtmomv
2
pzxkwxqkmlacrjkncsiobxytwkvonvwxszmzusczkogllixoanbjnxpmmjalgklgykzyjfwdzwynhtembpujdstaylvpjhwjacil
8
plgnlpdzeekvguskwylyjhwyjntnypwpbrbxvhhnrtiuukavxyqcaawqlvsjofaacpbbbulkhebrlqwhilwicgolmiorrzvwhtyj
9
gyapskhqhyprwrsmbhdhayxdyqewmmbsqbsbhdgrbjsprawvrzkvhkatanvplnrgdtixxkrlioacokjrbcnqvctzqmuwpulokmhe
7
otuofrtkzwalebsjqctycrmrrsutjfkvgishvqdxtgaahtgoavbuqwynfyptzgebexqrflzpvjyyolfgcffpahpmfrzhidtitdao
7
kpzqbftzmxuqmwnditbqvwhfckxwqdmbvrfuyomrrdxsjkthucgrcdqeygxsawsycvzlvnjizslnecgntvagarvwtzpalypyvmpa
9
pbdihigxurohmdgecrjxligjsgdectrbiguxgobxwgbzxmzorqqpqlhbupmdkianfajqjxzciyfgpzlrplezzqpcfmqswkbrylgv
5
caiikrgbhkoasppitbluijmtmuuodclifrjjkijcsxzdtkoekoptevljizxsafjrlyydskagfokodrborzxflrjeuuefkehcxrbm
3
