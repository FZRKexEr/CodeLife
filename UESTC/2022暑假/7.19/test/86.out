16384
16384
65980984
4096
2048
16384
16384
2048
1048576
603979768
